test/gemojione_test.rb

```diff
@@ -89,6 +89,36 @@
       assert_equal "I <img alt=\"❤\" class=\"emoji\" src=\"http://localhost:3000/2764.png\"> Emoji", replaced_string
     end
 
```

Comment:
These 2 first tests should test the new methods (`replace_unicode_moji_with_images` and `replace_named_moji_with_images` respectively) without the use of spritesheet option, like the 3rd one. That should be tested separately because it's a different and independent feature.